### ISSUE TITLE
Nextcloud windows explorer integration broke directory MRU links

### ISSUE THREAD

**Issue by Clapppa** [nextcloud/desktop#1599]
If integration for windows explorer is installed with nextcloud sync client, most recent used directory links on the taskbar stop working for directories inside Nextcloud sync directory (others keep working). You click them by left mouse button and nothing happens.

Client version: 2.6.1
Operating system: Windows 10
OS language: Russian

![Screenshot](https://i.ibb.co/j3p4bwc/scrnshot.png)

**Comment by Kurtoid** [comment]
Can confirm. They do work when opened from an existing explorer window from the quick access list, but doesnt work when launched from the Jumplist pictured above. Not sure if it's a windows bug or a Nextcloud bug

**Comment by alexbichlmaier** [comment]
Can confirm with all versions 2.5.3-final and higher. 2.5.3-rc2 and lower seem to be fine.

**Comment by Kurtoid** [comment]
So it does change w/ Nextcloud version? That would confirm the problem is Nextcloud, not Windows, right?

**Comment by alexbichlmaier** [comment]
It does very much seem like a Nextcloud bug.

After downgrading the client to 2.5.3-rc2 the jump list items work fine. They also work on more current versions of the client while the client isn't running.

I also discovered that only the left click on the jump list items is broken. Right clicking and choosing "Open" from the context menu also works regardless of Nextcloud version.

**Comment by ralf-f** [comment]
@alexbichlmaier: very usefull hint, the right clicking workaround! Exactly same behaviour here.

**Comment by Kurtoid** [comment]
I'm not sure how Windows handles the jumplist internally, but could #1671 be related? Both issues resolve themselves when the client isn't running.

**Comment by dbolton** [comment]
Owncloud had this issue earlier in 2019. I don't know if the underlying cause and fix would be the same or not. https://github.com/owncloud/client/issues/7004

**Comment by alexbichlmaier** [comment]
It's still broken in 2.6.3. 

Here's the client log, when clicking on the a pinned folder located in the Nextcloud folder:

```[OCC::SocketApi::slotNewConnection 	New connection QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "REGISTER_PATH:C:\\Users\\ab\\Nextcloud" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketApi::slotReadSocket 	Received SocketAPI message <-- "GET_STRINGS:CONTEXT_MENU_TITLE" from QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "GET_STRINGS:BEGIN" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "STRING:CONTEXT_MENU_TITLE:Share via Nextcloud" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "GET_STRINGS:END" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketApi::slotReadSocket 	Received SocketAPI message <-- "GET_MENU_ITEMS:C:\\Users\\ab\\Nextcloud\\Tools" from QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "GET_MENU_ITEMS:BEGIN" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "MENU_ITEM:SHARE::Freigabeoptionen" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "MENU_ITEM:COPY_PUBLIC_LINK::Öffentlichen Link kopieren" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "MENU_ITEM:COPY_PRIVATE_LINK::Kopiere internen Link" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "MENU_ITEM:OPEN_PRIVATE_LINK::Im Browser öffnen" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "GET_MENU_ITEMS:END" to QLocalSocket(0x2b19a8a8220)
[OCC::SocketApi::onLostConnection 	Lost connection  QLocalSocket(0x2b19a8a8220)
[OCC::SocketApi::slotNewConnection 	New connection QLocalSocket(0x2b19a8a85c0)
[OCC::SocketListener::sendMessage 	Sending SocketAPI message --> "REGISTER_PATH:C:\\Users\\ab\\Nextcloud" to QLocalSocket(0x2b19a8a85c0)
[OCC::SocketApi::onLostConnection 	Lost connection  QLocalSocket(0x2b19a8a85c0)
[OCC::SocketApi::slotReadSocket 	Received SocketAPI message <-- "OpenNewWindow:C:\\Users\\ab\\Nextcloud\\Tools" from QLocalSocket(0x2b19a8a85c0)
[OCC::SocketApi::slotReadSocket 	The command is not supported by this version of the client: "OpenNewWindow" with argument: "C:\\Users\\ab\\Nextcloud\\Tools"
```

The error message seems to suggest it's indeed related to the issue @dbolton referenced.

**Comment by misch7** [comment]
Merged #1829. Will get into the 2.6.4 release.
